src/main/groovy/edu/oregonstate/mist/osuevents/resources/EventsResource.groovy

```diff
@@ -289,6 +289,8 @@ class EventsResource extends Resource {
 
         if (!event.instances) {
             addBadRequest("At least one event instance is required.")
+        } else if (event.instances.contains(null)) {
```

Comment:
What do you think about adding this conditional check in the `else` block below it. My thinking is that if I have something like this:
```
"instances": [
          {
            "start": "2018-12-02T21:15:00Z",
            "end": "2017-12-02T21:30:00Z"
          },
          null
        ]
```
then it will just tell me an instance can not be null. It won't tell me that the end time can't be before the start time since one of the other instances is null.